doc/techref/encodings.md

```diff
@@ -1,3 +1,35 @@
+---
+file_format: mystnb
+---
+
+```{code-cell}
+---
+tags: [remove-input]
+---
+from IPython.display import display, Markdown
+from pygmt.encodings import charset
+
+
+def get_charset_mdtable(name):
```

Comment:
This function is modified from the original script at https://github.com/GenericMappingTools/pygmt/pull/3206#issue-2267029976